config/routes.rb

```diff
@@ -13,4 +13,10 @@
       post "downvote", to: "votes#downvote"
     end
   end
+
+  namespace :admin do
+    resources :users, only: [:index, :destroy]
+    resources :playlists, only: [:edit, :update, :destroy]
```

Comment:
Why would admins edit a user's playlist? Not sure how necessary that feature would be.